### PR TITLE
Prepare for DeepLinkDispatch v3.0.0-rc2/rc3 releases

### PR DESCRIPTION
**Pull request by felipecsl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,14 +144,50 @@ public class YourApplication extends Application {
...
+// "http://airbnb.com/users"
+// "http://airbnb.com/user/{id}"
+// "https://airbnb.com/users"
+// "https://airbnb.com/user/{id}"
+@AppDeepLink({ "/view_users" })
+@WebDeepLink({ "/users", "/user/{id}" })
+public class CustomPrefixesActivity extends AppCompatActivity {
+    //...
+}
+```
```
Comment (felipecsl) [inline README.md]: @kxfang @MichaelEvans does this look accurate to you?

Comment (kxfang) [inline README.md]: Yeah looks great, thanks!

**Comment by kxfang** [comment]
Let's fix #149 before release. I'll have a PR up shortly

**Comment by felipecsl** [comment]
Unfortunately I had already pushed rc2 before #150 got merged. I'm cutting rc3 now

**Comment by felipecsl** [comment]
3.0.0-rc3 uploaded. Will merge when it's live

**Pull request merged**